### DISCUSSION TITLE
beta.netbox.dev

### DISCUSSION THREAD

**Discussion by martinum4** [netbox-community/netbox#6971] [General]
Hey, can we get back beta.netbox.dev with the current testing beta version please? I know it was working way before (a few months) but i don't remember when it stopped exactly.

It would be neat to test the general look and feel and the early catching of bugs.

Kind Regards
Martin

**Comment by markkrj** [comment]
There is https://demo.netbox.dev and https://beta-demo.netbox.dev

**Reply by martinum4** [reply]
> https://beta-demo.netbox.dev

Ah nice, i didn't know that one existed, thanks
